package-lock.json

```diff
@@ -12199,7 +12190,7 @@
       "version": "18.2.0",
       "resolved": "https://registry.npmjs.org/react/-/react-18.2.0.tgz",
       "integrity": "sha512-/3IjMdb2L9QbBdWiW5e3P2/npwMBaU9mHCSCUzNln0ZCYbcfTsGbTJrU/kGemdH2IWmB2ioZ+zkxtmq6g09fGQ==",
-      "dev": true,
+      "peer": true,
```

Comment:
this is still using `18.2.0` without a specific devDep afaict, although the `"peer": true` part seems new